packages/amplify-cli/src/project-config-version-check.ts

```diff
@@ -65,14 +65,14 @@ async function checkLambdaCustomResourceNodeVersion(context: Context, projectPat
   const filesToUpdate: string[] = [];
 
   if (fs.existsSync(backendDirPath)) {
-    const globOptions: glob.IOptions = {
+    const globOptions: GlobOptionsWithFileTypesFalse = {
```

Comment:
that type name is funny.

Reply:
Casting as that type gives me the right return type of globSync. I thought it looks lame too